## Actual Cost of Living

In [1]:
import pandas as pd

### Exploratory Analysis

In [21]:
# reading
url = "https://drive.google.com/file/d/1yFlPVSCG0_0a5SI4RDfWiAH4Me6Acv2O/view?usp=sharing"
path = 'https://drive.google.com/uc?export=download&id='+url.split('/')[-2]
data = pd.read_csv(path)

In [22]:
data

Unnamed: 0                city      country     x1     x2     x3     x4  \
0              0               Delhi        India   4.90  22.04   4.28   1.84   
1              1            Shanghai        China   5.59  40.51   5.59   1.12   
2              2             Jakarta    Indonesia   2.54  22.25   3.50   2.02   
3              3              Manila  Philippines   3.54  27.40   3.54   1.24   
4              4               Seoul  South Korea   7.16  52.77   6.03   3.02   
...          ...                 ...          ...    ...    ...    ...    ...   
4869        4869        Peterborough    Australia    NaN    NaN    NaN    NaN   
4870        4870          Georgetown    Australia    NaN    NaN    NaN    NaN   
4871        4871  Ixtapa Zihuatanejo       Mexico   5.19  31.13  12.97   0.99   
4872        4872             Iqaluit       Canada  29.78  74.61  13.77   6.70   
4873        4873              Neiafu        Tonga    NaN  29.53  10.55  10.55   

        x5    x6    x7  ...     x47      x48     x49      x50      x51  \
0     3.67  1.78  0.48  ...   36.26   223.87  133.38   596.16   325.82   
1     4.19  3.96  0.52  ...  121.19  1080.07  564.30  2972.57  1532.23   
2     3.18  2.19  0.59  ...   80.32   482.85  270.15  1117.69   584.37   
3     1.90  2.91  0.93  ...   61.82   559.52  281.78  1754.40   684.81   
4     4.52  3.86  1.46  ...  108.30   809.83  583.60  2621.05  1683.74   
...    ...   ...   ...  ...     ...      ...     ...      ...      ...   
4869   NaN   NaN   NaN  ...     NaN      NaN     NaN      NaN      NaN   
4870   NaN   NaN   NaN  ...     NaN      NaN     NaN      NaN      NaN   
4871   NaN  1.82  0.62  ...  103.78   415.11  259.44   518.89   415.11   
4872  8.93  3.72  3.54  ...     NaN      NaN     NaN  2978.11  2978.11   
4873   NaN   NaN  2.11  ...     NaN      NaN     NaN      NaN      NaN   

           x52       x53      x54   x55  data_quality  
0      2619.46   1068.90   586.35  7.96             1  
1     17333.09   9174.88  1382.83  5.01             1  
2      2694.05   1269.44   483.19  9.15             1  
3      3536.04   2596.44   419.02  7.80             1  
4     21847.94  10832.90  2672.23  3.47             1  
...        ...       ...      ...   ...           ...  
4869       NaN       NaN      NaN   NaN             0  
4870       NaN       NaN      NaN   NaN             0  
4871       NaN       NaN      NaN   NaN             0  
4872       NaN       NaN      NaN  6.53             0  
4873       NaN       NaN      NaN   NaN             0  

[4874 rows x 59 columns]

In [23]:
data.columns

Index(['Unnamed: 0', 'city', 'country', 'x1', 'x2', 'x3', 'x4', 'x5', 'x6',
       'x7', 'x8', 'x9', 'x10', 'x11', 'x12', 'x13', 'x14', 'x15', 'x16',
       'x17', 'x18', 'x19', 'x20', 'x21', 'x22', 'x23', 'x24', 'x25', 'x26',
       'x27', 'x28', 'x29', 'x30', 'x31', 'x32', 'x33', 'x34', 'x35', 'x36',
       'x37', 'x38', 'x39', 'x40', 'x41', 'x42', 'x43', 'x44', 'x45', 'x46',
       'x47', 'x48', 'x49', 'x50', 'x51', 'x52', 'x53', 'x54', 'x55',
       'data_quality'],
      dtype='object')

In [24]:
data = data.drop(columns=['Unnamed: 0'])

In [25]:
data

city      country     x1     x2     x3     x4    x5    x6  \
0                  Delhi        India   4.90  22.04   4.28   1.84  3.67  1.78   
1               Shanghai        China   5.59  40.51   5.59   1.12  4.19  3.96   
2                Jakarta    Indonesia   2.54  22.25   3.50   2.02  3.18  2.19   
3                 Manila  Philippines   3.54  27.40   3.54   1.24  1.90  2.91   
4                  Seoul  South Korea   7.16  52.77   6.03   3.02  4.52  3.86   
...                  ...          ...    ...    ...    ...    ...   ...   ...   
4869        Peterborough    Australia    NaN    NaN    NaN    NaN   NaN   NaN   
4870          Georgetown    Australia    NaN    NaN    NaN    NaN   NaN   NaN   
4871  Ixtapa Zihuatanejo       Mexico   5.19  31.13  12.97   0.99   NaN  1.82   
4872             Iqaluit       Canada  29.78  74.61  13.77   6.70  8.93  3.72   
4873              Neiafu        Tonga    NaN  29.53  10.55  10.55   NaN   NaN   

        x7    x8  ...     x47      x48     x49      x50      x51       x52  \
0     0.48  0.19  ...   36.26   223.87  133.38   596.16   325.82   2619.46   
1     0.52  0.32  ...  121.19  1080.07  564.30  2972.57  1532.23  17333.09   
2     0.59  0.27  ...   80.32   482.85  270.15  1117.69   584.37   2694.05   
3     0.93  0.51  ...   61.82   559.52  281.78  1754.40   684.81   3536.04   
4     1.46  0.78  ...  108.30   809.83  583.60  2621.05  1683.74  21847.94   
...    ...   ...  ...     ...      ...     ...      ...      ...       ...   
4869   NaN   NaN  ...     NaN      NaN     NaN      NaN      NaN       NaN   
4870   NaN   NaN  ...     NaN      NaN     NaN      NaN      NaN       NaN   
4871  0.62  0.42  ...  103.78   415.11  259.44   518.89   415.11       NaN   
4872  3.54  4.10  ...     NaN      NaN     NaN  2978.11  2978.11       NaN   
4873  2.11  2.11  ...     NaN      NaN     NaN      NaN      NaN       NaN   

           x53      x54   x55  data_quality  
0      1068.90   586.35  7.96             1  
1      9174.88  1382.83  5.01             1  
2      1269.44   483.19  9.15             1  
3      2596.44   419.02  7.80             1  
4     10832.90  2672.23  3.47             1  
...        ...      ...   ...           ...  
4869       NaN      NaN   NaN             0  
4870       NaN      NaN   NaN             0  
4871       NaN      NaN   NaN             0  
4872       NaN      NaN  6.53             0  
4873       NaN      NaN   NaN             0  

[4874 rows x 58 columns]

#### Selection of 15 Cities

In [26]:
# List of selected cities
selected_cities = [
    'Berlin', 'Madrid', 'Paris', 'Vienna', 'Hamburg', 'Bucharest',
    'Budapest', 'Barcelona', 'Munich', 'Prague', 'Milan', 'Sofia',
    'Cologne', 'Stockholm', 'Amsterdam'
]

In [27]:
# Filter the DataFrame to include only the selected cities
filtered_data = data[data['city'].isin(selected_cities)]

In [28]:
# Filter Barcelona to only include Spain
filtered_data = filtered_data[(filtered_data['city'] != 'Barcelona') | (filtered_data['country'] == 'Spain')]


In [29]:
filtered_data

city      country     x1     x2     x3    x4    x5    x6    x7  \
29       Paris       France  15.58  62.32  10.39  7.27  6.23  3.80  2.84   
81      Madrid        Spain  12.46  51.93   8.31  3.64  3.64  2.20  2.33   
114  Barcelona        Spain  14.54  62.32   9.35  3.12  3.64  2.32  2.15   
144     Berlin      Germany  12.46  62.32   9.35  4.15  4.15  3.33  2.47   
247     Vienna      Austria  15.58  62.32   9.35  4.67  4.21  3.80  2.99   
255  Bucharest      Romania  10.56  42.26   5.49  2.11  2.54  2.53  1.85   
256    Hamburg      Germany  14.02  62.32   9.35  4.67  4.15  3.51  2.69   
270   Budapest      Hungary   7.93  38.35   5.63  1.92  2.05  1.79  1.09   
307     Munich      Germany  15.58  62.32   9.35  4.15  4.15  3.53  3.14   
326      Milan        Italy  20.77  83.10   9.61  6.23  5.19  1.84  2.73   
341      Sofia     Bulgaria  10.62  37.17   6.37  1.86  2.65  1.89  1.22   
396    Cologne      Germany  12.98  57.13   9.34  3.64  3.01  2.81  2.29   
449  Stockholm       Sweden  12.38  76.21   8.57  6.67  6.67  3.87  2.28   
456  Amsterdam  Netherlands  19.22  83.10  10.39  5.90  5.19  3.54  2.74   

       x8  ...     x47      x48      x49      x50      x51       x52      x53  \
29   2.40  ...  136.99  1325.47   978.26  2968.63  2018.27  12811.70  9084.89   
81   1.66  ...   95.09  1041.06   827.49  1781.48  1287.54   5469.86  3667.02   
114  1.49  ...  105.06  1075.89   840.13  1772.54  1368.73   5030.24  3059.61   
144  1.91  ...  108.49  1307.95   953.36  2410.40  1722.39   8481.50  5598.57   
247  2.31  ...  123.29   983.06   690.86  1932.98  1328.32   9095.45  5760.31   
255  1.60  ...   91.98   461.55   333.19   872.84   560.47   2490.88  1518.71   
256  2.19  ...  146.58  1107.94   832.60  2114.05  1587.26   9125.69  5853.80   
270  0.91  ...   86.60   475.62   368.37   881.43   633.50   3388.39  2255.78   
307  2.88  ...  119.40  1407.34  1077.20  2563.69  1932.74  12777.09  9381.58   
326  1.30  ...  157.40  1304.32   856.00  2689.48  1746.49   8983.21  4691.45   
341  0.83  ...   86.10   437.61   330.07   821.13   564.04   2531.65  1627.04   
396  1.81  ...  115.21   886.09   681.64  1649.93  1206.37   5933.55  4596.23   
449  2.05  ...  122.69  1365.56  1016.12  2274.50  1547.31  10986.50  6504.01   
456  2.48  ...  134.73  1769.11  1512.73  2970.60  2175.95   9626.48  6437.84   

         x54   x55  data_quality  
29   2741.22  1.67             1  
81   2022.60  2.29             1  
114  1924.32  2.11             1  
144  3086.47  2.16             1  
247  2662.28  2.15             1  
255   933.79  6.40             1  
256  3127.85  1.94             1  
270   941.34  8.13             1  
307  3435.65  2.00             1  
326  1868.62  1.66             1  
341   996.71  3.17             1  
396  2444.84  2.67             1  
449  2981.76  2.43             1  
456  3606.11  3.04             1  

[14 rows x 58 columns]

In [30]:
ordered_data_list = [filtered_data[filtered_data['city'] == city] for city in selected_cities]
ordered_data = pd.concat(ordered_data_list)

In [ ]:
ordered_data

In [32]:
# Mapping of current column names to new suitable names
column_mapping = {
    'x1': 'Meal_Inexpensive_Restaurant', 'x2': 'Meal_For_2_Mid_Range_Restaurant', 'x3': 'McMeal_At_McDonalds',
    'x4': 'Domestic_Beer_Restaurant', 'x5': 'Imported_Beer_Restaurant', 'x6': 'Cappuccino',
    'x7': 'Coke_Pepsi', 'x8': 'Water_o.33L', 'x9': 'Milk_1L', 'x10': 'Bread_500g',
    'x11': 'Rice_1kg', 'x12': 'Eggs_12', 'x13': 'Cheese_1kg', 'x14': 'Chicken_Fillets_1kg', 'x15': 'Beef_1kg',
    'x16': 'Apples_1kg', 'x17': 'Banana_1kg', 'x18': 'Oranges_1kg', 'x19': 'Tomato_1kg', 'x20': 'Potato_1kg',
    'x21': 'Onion_1kg', 'x22': 'Lettuce_1head', 'x23': 'Water_1.5L_Bottle', 'x24': 'Bottle_Of_Wine',
    'x25': 'Domestic_Beer_Market', 'x26': 'Imported_Beer_Market', 'x27': 'Cigarettes_20_Pack',
    'x28': 'One_Way_Ticket_Transport', 'x29': 'Monthly_Pass_Transport', 'x30': 'Taxi_Start_Tariff',
    'x31': 'Taxi_1km_Tariff', 'x32': 'Taxi_1hour_Waiting_Tariff', 'x33': 'Gasoline_1L',
    'x34': 'Volkswagen_Golf', 'x35': 'Toyota_Corolla', 'x36': 'Basic_Utilities_85m2',
    'x37': 'Prepaid_Mobile_Tariff', 'x38': 'Internet_60Mbps', 'x39': 'Fitness_Club_Fee',
    'x40': 'Tennis_Court_Rent', 'x41': 'Cinema_1Seat', 'x42': 'Preschool_Monthly_Fee',
    'x43': 'International_School_Yearly', 'x44': 'Jeans_Levis_501', 'x45': 'Summer_Dress_Chain_Store',
    'x46': 'Nike_Running_Shoes', 'x47': 'Men_Leather_Shoes', 'x48': 'Apartment_1bed_City_Center',
    'x49': 'Apartment_1bed_Outside_Center', 'x50': 'Apartment_3bed_City_Center', 'x51': 'Apartment_3bed_Outside_Center',
    'x52': 'Price_per_m2_City_Center', 'x53': 'Price_per_m2_Outside_Center'
}


In [33]:
# Rename the columns in the DataFrame
ordered_data.rename(columns=column_mapping, inplace=True)


In [38]:
ordered_data.head()

city  country  Meal_Inexpensive_Restaurant  \
144   Berlin  Germany                        12.46   
81    Madrid    Spain                        12.46   
29     Paris   France                        15.58   
247   Vienna  Austria                        15.58   
256  Hamburg  Germany                        14.02   

     Meal_For_2_Mid_Range_Restaurant  McMeal_At_McDonalds  \
144                            62.32                 9.35   
81                             51.93                 8.31   
29                             62.32                10.39   
247                            62.32                 9.35   
256                            62.32                 9.35   

     Domestic_Beer_Restaurant  Imported_Beer_Restaurant  Cappuccino  \
144                      4.15                      4.15        3.33   
81                       3.64                      3.64        2.20   
29                       7.27                      6.23        3.80   
247                      4.67                      4.21        3.80   
256                      4.67                      4.15        3.51   

     Coke_Pepsi  Water_o.33L  ...  Men_Leather_Shoes  \
144        2.47         1.91  ...             108.49   
81         2.33         1.66  ...              95.09   
29         2.84         2.40  ...             136.99   
247        2.99         2.31  ...             123.29   
256        2.69         2.19  ...             146.58   

     Apartment_1bed_City_Center  Apartment_1bed_Outside_Center  \
144                     1307.95                         953.36   
81                      1041.06                         827.49   
29                      1325.47                         978.26   
247                      983.06                         690.86   
256                     1107.94                         832.60   

     Apartment_3bed_City_Center  Apartment_3bed_Outside_Center  \
144                     2410.40                        1722.39   
81                      1781.48                        1287.54   
29                      2968.63                        2018.27   
247                     1932.98                        1328.32   
256                     2114.05                        1587.26   

     Price_per_m2_City_Center  Price_per_m2_Outside_Center      x54   x55  \
144                   8481.50                      5598.57  3086.47  2.16   
81                    5469.86                      3667.02  2022.60  2.29   
29                   12811.70                      9084.89  2741.22  1.67   
247                   9095.45                      5760.31  2662.28  2.15   
256                   9125.69                      5853.80  3127.85  1.94   

     data_quality  
144             1  
81              1  
29              1  
247             1  
256             1  

[5 rows x 58 columns]

In [39]:
# Columns for each category
food_columns = ['city', 'country', 'Meal_Inexpensive_Restaurant', 'Meal_For_2_Mid_Range_Restaurant', 'McMeal_At_McDonalds',
                'Domestic_Beer_Restaurant', 'Imported_Beer_Restaurant', 'Cappuccino', 'Coke_Pepsi',
                'Water_o.33L', 'Milk_1L', 'Bread_500g', 'Rice_1kg', 'Eggs_12', 'Cheese_1kg', 'Chicken_Fillets_1kg',
                'Beef_1kg', 'Apples_1kg', 'Banana_1kg', 'Oranges_1kg', 'Tomato_1kg', 'Potato_1kg', 'Onion_1kg',
                'Lettuce_1head', 'Water_1.5L_Bottle', 'Bottle_Of_Wine', 'Domestic_Beer_Market', 'Imported_Beer_Market',
                'Cigarettes_20_Pack']
transportation_columns = ['city', 'country', 'One_Way_Ticket_Transport', 'Monthly_Pass_Transport', 'Taxi_Start_Tariff',
                          'Taxi_1km_Tariff', 'Taxi_1hour_Waiting_Tariff', 'Gasoline_1L']
basic_needs_columns = ['city', 'country', 'Basic_Utilities_85m2',
                       'Prepaid_Mobile_Tariff', 'Internet_60Mbps', 'Fitness_Club_Fee', 'Tennis_Court_Rent',
                       'Cinema_1Seat', 'Preschool_Monthly_Fee', 'International_School_Yearly']
housing_columns = ['city', 'country', 'Apartment_1bed_City_Center', 'Apartment_1bed_Outside_Center',
                   'Apartment_3bed_City_Center', 'Apartment_3bed_Outside_Center', 'Price_per_m2_City_Center',
                   'Price_per_m2_Outside_Center']



In [ ]:
#food_columns

In [40]:
# Create DataFrames for each category
food_data = ordered_data[food_columns].copy()
transportation_data = ordered_data[transportation_columns].copy()
basic_needs_data = ordered_data[basic_needs_columns].copy()
housing_data = ordered_data[housing_columns].copy()

In [41]:
avg_ex_rate = 0.88  # Replace with the actual average exchange rate

In [42]:
# Function to convert USD to Euros based on average exchange rate
def usd_to_eur(price_usd):
    return round(price_usd * avg_ex_rate,2)

In [43]:
# Convert USD prices to Euros for each category
for col in food_columns[2:]:  # Start from index 2 to skip 'city' and 'country'
    food_data[col] = food_data[col].apply(usd_to_eur)

for col in transportation_columns[2:]:
    transportation_data[col] = transportation_data[col].apply(usd_to_eur)

for col in basic_needs_columns[2:]:
    basic_needs_data[col] = basic_needs_data[col].apply(usd_to_eur)

for col in housing_columns[2:]:
    housing_data[col] = housing_data[col].apply(usd_to_eur)

In [44]:
# Add a 'Category' column to each DataFrame
food_data['Category'] = 'Food'
transportation_data['Category'] = 'Transportation'
basic_needs_data['Category'] = 'Basic Needs'
housing_data['Category'] = 'Housing'

In [ ]:
# Combine all DataFrames into one
#combined_data = pd.concat([food_data, transportation_data, basic_needs_data, housing_data])


In [ ]:
housing_data.isna().sum()

In [46]:
basic_needs_data

city      country  Basic_Utilities_85m2  Prepaid_Mobile_Tariff  \
144     Berlin      Germany                266.29                   0.10   
81      Madrid        Spain                144.06                   0.15   
29       Paris       France                174.06                   0.15   
247     Vienna      Austria                187.84                   0.06   
256    Hamburg      Germany                249.08                   0.11   
255  Bucharest      Romania                103.24                   0.07   
270   Budapest      Hungary                130.83                   0.07   
114  Barcelona        Spain                148.20                   0.20   
307     Munich      Germany                266.04                   0.10   
326      Milan        Italy                189.28                   0.14   
341      Sofia     Bulgaria                118.36                   0.17   
396    Cologne      Germany                243.28                   0.11   
449  Stockholm       Sweden                 86.50                   0.12   
456  Amsterdam  Netherlands                244.12                   0.11   

     Internet_60Mbps  Fitness_Club_Fee  Tennis_Court_Rent  Cinema_1Seat  \
144            35.12             35.59              17.16         10.96   
81             32.74             37.50              10.64          8.23   
29             26.86             32.70              12.03         10.96   
247            29.62             25.24              18.28         10.06   
256            32.74             48.02              25.02         13.71   
255             7.27             39.68              15.78          6.51   
270            11.18             38.70               9.92          5.17   
114            34.82             39.60              20.59          8.23   
307            33.10             35.25              21.79         10.96   
326            26.06             68.57              23.01          9.14   
341            11.26             26.00              12.78          7.00   
396            30.26             28.23              25.21         10.96   
449            29.48             36.30              29.55         12.58   
456            43.01             37.44              26.00         11.88   

     Preschool_Monthly_Fee  International_School_Yearly     Category  
144                 144.07                      7769.45  Basic Needs  
81                  451.89                      8394.06  Basic Needs  
29                  686.93                     15016.58  Basic Needs  
247                 246.17                     10351.65  Basic Needs  
256                 530.15                     21414.96  Basic Needs  
255                 414.64                      7224.02  Basic Needs  
270                 312.80                      7674.82  Basic Needs  
114                 440.70                      9892.42  Basic Needs  
307                 586.25                     14767.03  Basic Needs  
326                 698.98                     14451.48  Basic Needs  
341                 382.08                      6291.32  Basic Needs  
396                 500.60                     10603.01  Basic Needs  
449                 116.67                      7005.78  Basic Needs  
456                1631.78                     12671.63  Basic Needs

In [47]:
food_data
food_data.to_csv('./food_costs.csv',index=False)

In [48]:
transportation_data.to_csv('./transport_costs.csv',index=False)

In [49]:
basic_needs_data.to_csv('./basic_needs.csv',index=False)

In [50]:
housing_data.to_csv('./housing_costs.csv',index=False)

In [51]:
from google.colab import files
#files.download('./food_costs.csv')
#files.download('./transport_costs.csv')
#files.download('./basic_needs.csv')
#files.download('./housing_costs.csv')

<IPython.core.display.Javascript object>

<IPython.core.display.Javascript object>

<IPython.core.display.Javascript object>

<IPython.core.display.Javascript object>

<IPython.core.display.Javascript object>

<IPython.core.display.Javascript object>

<IPython.core.display.Javascript object>

<IPython.core.display.Javascript object>

#### **Note** Data is from 2022 and Prague city is not there in dataset so we've skipped it

### Categories and Column Mapping:

####**Food**
* x1: Meal, Inexpensive Restaurant (Eur)
* x2: Meal for 2 People, Mid-range Restaurant, Three-course (Eur)
* x3: McMeal at McDonalds (or Equivalent Combo Meal) (Eur)
* x4: Domestic Beer (0.5 liter draught, in restaurants) (Eur)
* x5: Imported Beer (0.33 liter bottle, in restaurants) (Eur)
* x6: Cappuccino (regular, in restaurants) (Eur)
* x7: Coke/Pepsi (0.33 liter bottle, in restaurants) (Eur)
* x8: Water (0.33 liter bottle, in restaurants) (Eur)
* x9: Milk (regular), (1 liter) (Eur)
* x10: Loaf of Fresh White Bread (500g) (Eur)
* x11: Rice (white), (1kg) (Eur)
* x12: Eggs (regular) (12) (Eur)
* x13: Local Cheese (1kg) (Eur)
* x14: Chicken Fillets (1kg) (Eur)
* x15: Beef Round (1kg) (or Equivalent Back Leg Red Meat) (Eur)
* x16: Apples (1kg) (Eur)
* x17: Banana (1kg) (Eur)
* x18: Oranges (1kg) (Eur)
* x19: Tomato (1kg) (Eur)
* x20: Potato (1kg) (Eur)
* x21: Onion (1kg) (Eur)
* x22: Lettuce (1 head) (Eur)
* x23: Water (1.5 liter bottle, at the market) (Eur)
* x24: Bottle of Wine (Mid-Range, at the market) (Eur)
* x25: Domestic Beer (0.5 liter bottle, at the market) (Eur)
* x26: Imported Beer (0.33 liter bottle, at the market) (Eur)
* x27: Cigarettes 20 Pack (Marlboro) (Eur)

#### **Transportation**
* x28: One-way Ticket (Local Transport) (Eur)
* x29: Monthly Pass (Regular Price) (Eur)
* x30: Taxi Start (Normal Tariff) (Eur)
* x31: Taxi 1km (Normal Tariff) (Eur)
* x32: Taxi 1hour Waiting (Normal Tariff) (Eur)
* x33: Gasoline (1 liter) (Eur)

####**Basic Needs**
* x34: Volkswagen Golf 1.4 90 KW Trendline (Or Equivalent New Car) (Eur)
* x35: Toyota Corolla Sedan 1.6l 97kW Comfort (Or Equivalent New Car) (Eur)
* x36: Basic (Electricity, Heating, Cooling, Water, Garbage) for 85m2 Apartment (Eur)
* x37: 1 min. of Prepaid Mobile Tariff Local (No Discounts or Plans) (Eur)
* x38: Internet (60 Mbps or More, Unlimited Data, Cable/ADSL) (Eur)
* x39: Fitness Club, Monthly Fee for 1 Adult (Eur)
* x40: Tennis Court Rent (1 Hour on Weekend) (Eur)
* x41: Cinema, International Release, 1 Seat (Eur)
* x42: Preschool (or Kindergarten), Full Day, Private, Monthly for 1 Child (Eur)
* x43: International Primary School, Yearly for 1 Child (Eur)

####**Housing**
* x48: Apartment (1 bedroom) in City Centre (Eur)
* x49: Apartment (1 bedroom) Outside of Centre (Eur)
* x50: Apartment (3 bedrooms) in City Centre (Eur)
* x51: Apartment (3 bedrooms) Outside of Centre (Eur)
* x52: Price per Square Meter to Buy Apartment in City Centre (Eur)
* x53: Price per Square Meter to Buy Apartment Outside of Centre (Eur)